internal/services/containerapps/container_app_resource_test.go

```diff
@@ -195,6 +210,28 @@ func TestAccContainerAppResource_completeUpdate(t *testing.T) {
 	})
 }
 
+func TestAccContainerAppResource_removeDaprAppPort(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_container_app", "test")
+	r := ContainerAppResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data, "rev1"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.completeUpdate_withNoDaprAppPort(data, "rev2"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("dapr.app_port").IsEmpty(),
```

Comment:
I think the path here would need to be 
```suggestion
				check.That(data.ResourceName).Key("dapr.0.app_port").IsEmpty(),
```
for the test to evaluate correctly.